### PR TITLE
Update configuration

### PR DESCRIPTION
**Pull request by JuMastro**
* Add Travis configuraton included codeClimate & tests

* Add .gittatributes

* Add .editorconfig

* Add eslint

Add eslint and plugins (import, node, promise) + standard set
Add .eslintrc.js & .eslintignore

* Add package.json with inital configuration + .lock

* Add CHANGELOG.md

### CODE REVIEW

**Comment by JuMastro** [comment]
# TODO:

## Travis
- Should link to [travis.com](https://travis-ci.com)
- Should set env vars (NPM_API_KEY, NPM_EMAIL & CC_TEST_REPORTER_ID)

CC_TEST_REPORTER_ID can be found from "Analysis > Test coverage" from codeClimate project settings.

## CodeClimate
- Should link to [codeClimate](https://codeclimate.com)

image
image
image

**Comment by JuMastro** [comment]
# Important
- Should add branch security for `master`, wait for review + travis validation + (codeClimate validation?!)

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "cherry",
+  "version": "0.0.1",
+  "description": "A light framework to quickly create a web server",
+  "main": "./src/index.js",
```
Comment (Mystilund) [inline package.json]: Wrong, this file doesn't exist

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (Mystilund) [inline .editorconfig]: File to delete

**Review by Mystilund — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (JuMastro) [reply package.json]: It's update to `./src/cherry.js`

File: .editorconfig (hunk 2 above)
Comment (JuMastro) [reply .editorconfig]: This file is now deleted

**Review by Mystilund — APPROVED** [review]

**Pull request merged**